scripts/deploy.sh

```diff
@@ -56,12 +56,15 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # remove unused variables file
+    rm /etc/openstack_deploy/user_variables.yml
```

Comment:
That would explain the fails, I'll probably need to still install it to a new location and change what points to it.